lexer/lexer_test.go

```diff
@@ -2528,7 +2528,7 @@ s: >-3
 					CharacterType: token.CharacterTypeIndicator,
 					Indicator:     token.BlockScalarIndicator,
 					Value:         "|",
-					Origin: "\n|  		  ", //nolint:gci,gofmt
+					Origin:        "\n|  		  ", //nolint:gci,gofmt
```

Comment:
My `gofmt` ( maybe `gofmt` built by Go 1.22.8 ) changes to the current state, so I don't accept this changes 🤔 
I think it's a difference depending on the version of gofmt.

Reply:
No, it wasn't. I think it was probably due to goimports. I'll look into it a bit more.

Reply:
Sorry, the problem was that my `goimports` were very old. I updated it and the problem was solved 😄

Reply:
Please remove `//nolint:gci,gofmt` comment 🙏